app/Models/Observers/UserObserver.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: ToothlessRebel
+ * Date: 30-Dec-16
```

Comment:
You're right, I'm not going back and fixing them all but I **will** update settings to make it more standard.